curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efc4f528d6a74d05e68af74.md

```diff
@@ -25,14 +25,12 @@ Your checkbox should still have an `id` attribute with the value `loving`.  Expe
 assert.equal($('input[type="checkbox"]')[0].id, 'loving');
 ```
 
-The text ` Loving` should no longer be located directly to the right of your checkbox. It should be wrapped in a `label` element. Make sure there is a space between the two elements.
+The text `Loving` should be wrapped in a `label` element.
```

Comment:
If we are changing the test hint, we should change them across all localization to be consistent in all messages.